my-diploma/src/Components/loading/Spinner.js

```diff
@@ -0,0 +1,24 @@
+import CircularProgress, {
+  circularProgressClasses,
+} from "@mui/material/CircularProgress";
+
+const Spinner = (props) => {
+  return (
+    <CircularProgress
+      variant="indeterminate"
+      disableShrink
+      sx={{
+        color: "red",
+        animationDuration: "700ms",
+        [`& .${circularProgressClasses.circle}`]: {
+          strokeLinecap: "round",
+        },
+      }}
+      size={60}
+      thickness={10}
+      {...props}
+    />
+  );
+};
+
+export default Spinner;
```

Comment:
1) Components folder must not be from capital letter.
2) loading folder must be from capital letter, because it contains component
3) there is no sense to name component file as a folder, use index.js naming, because folder already tells you which component inside of it